spec/lexers/ssh_spec.rb

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+describe Rouge::Lexers::SSH do
```

Comment:
The lexer spec should also check the registered filenames are working correctly. See [the JavaScript lexer's spec](https://github.com/rouge-ruby/rouge/blob/29ad1baae71908336e9325efa624f88010ed7b48/spec/lexers/javascript_spec.rb) for examples.